databricks/koalas/base.py

```diff
@@ -599,6 +602,9 @@ def notnull(self):
         >>> ser.rename("a").to_frame().set_index("a").index.notna()
         Index([True, True, False], dtype='object', name='a')
         """
+        from databricks.koalas.indexes import MultiIndex
+        if isinstance(self, MultiIndex):
+            raise NotImplementedError("isna is not defined for MultiIndex")
```

Comment:
Let's just fix `isna` to `notnull` here.

Reply:
at the first time, i just mimic pandas', but i think also it is better. thanks!